SQL console/src/com/serotonin/m2m2/db/dao/SqlConsole.java

```diff
@@ -68,4 +66,26 @@ else if (meta.getColumnType(i + 1) == Types.LONGVARBINARY
             return result;
         });
     }
+
+
+    private static Object readObjectFromBlob(Blob blob) throws SQLException {
+        Object o;
+
```

Comment:
Please check your formatting settings in your IDE.  That is all explained in the IDE setup doc on confluence.  These extra spaces between lines are not required nor desired.  Please remove them.

Reply:
I removed the extra spaces.